src/components/FormAddBook.js

```diff
@@ -32,18 +31,6 @@ function FormAddBook() {
         <input type="text" name="author" placeholder="author" id="author" ref={authorInputRef} required />
         <input type="text" name="category" placeholder="Category" id="category" ref={categoryInputRef} required />
         <button type="submit">Add Book</button>
```

Comment:
- [ ] This component should have their inputs that updates the local state every time they update (onChange)